### PR TITLE
Trajectorystop model

### PR DESCRIPTION
**Pull request by REYNALopez12**
The trajectory stop table model was created

### CODE REVIEW

File: app/models/service.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Service < ApplicationRecord
+    has_many : trajectory 
+
+     validates :name, presence: true
```
Comment (jesusMava) [inline app/models/service.rb]: only one validate?

File: app/models/service.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Service < ApplicationRecord
+    has_many : trajectory
```
Comment (jesusMava) [inline app/models/service.rb]: should it be in plural, shouldn't it?

**Review by jesusMava — CHANGES_REQUESTED** [review]

File: app/models/service.rb (hunk 2 above)
Comment (mtopete1996) [inline app/models/service.rb]:
```suggestion
    has_many :trajectories
```

File: app/models/unit.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class Unit < ApplicationRecord
+    has_one : trajectory
+    has_many : type
```
Comment (mtopete1996) [inline app/models/unit.rb]:
```suggestion
    has_one :trajectory
    has_many :types
```

File: lib/trajectories/route_10.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (mtopete1996) [inline lib/trajectories/route_10.json]: This file shouldn't be here

File: lib/trajectories/route_21.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (mtopete1996) [inline lib/trajectories/route_21.json]: This what?

**Review by mtopete1996 — CHANGES_REQUESTED** [review]

File: app/models/service.rb (hunk 1 above)
Comment (REYNALopez12) [reply app/models/service.rb]: yes

**Review by mtopete1996 — APPROVED** [review]

**Pull request merged**